pex/pex.py

```diff
@@ -585,6 +585,10 @@ def execute_interpreter(self):
                 module = args[1]
                 sys.argv = args[1:]
                 return self.execute_module(module, alter_sys=True)
+            elif arg == "-s":
```

Comment:
The execute_interpreter mode is intended to act just like `python` as much as possible. The `-s` option means something else for `python` and `python` doesn't even have the concept of a console script - that's a higher level thing added by `setuptools` distributions.

You may know this stuff, so forgive me if you do - but just to run down all the options today:
1. If you just want to always use 1 console script, build the pex with `-c` / `--script` / `--console-script`.
2. If you want to use an add hoc script on the CLI just use `PEX_SCRIPT=foo ./my.pex ...`.
3. As an alternative to 2, build your pex with `--include-tools` and then run `PEX_TOOLS=1 ./my.pex venv create/a/venv/from/the/my.pex/right/here` and then source the activate script and run any console script you want.

Reply:
I have a use case for a project where there are multiples console-script defined, and I want a user to be able to choose which one their wants at runtime.
Using PEX_SCRIPT works fine, I just find that an option parameter would have a better usability in this case.